src/Sulu/Bundle/TestBundle/Testing/TestUserProvider.php

```diff
@@ -141,7 +140,7 @@ public function refreshUser(UserInterface $user)
      */
     public function supportsClass($class)
     {
-        return $class instanceof UserInterface;
+        return is_subclass_of($class, UserInterface::class);
```

Comment:
@danrot Looks like tests did call the false user provider before for supportsClass. This was fixed in symfony 3.4.37 and so I copied the supportsClass function from our UserProvider to the TestUserProvider. As `$class instanceof UserInterface` does nothing.

Reply:
In 2.0 we wrap the origina UserProvider so the fix looks different there #4999